data_pre_processing.md

```diff
@@ -11,7 +11,7 @@ Dataset used during this session can be found in the following location within a
 
```

Comment:
We should probably move the test data to /projects too. Also, maybe remove the Dec14 tag? That would make it more independent.

Reply:
ok - we can  remove tag, then we have to rename the files on Abel. - has to be consistent. Can rename when done.
So: pending